content/documentation/runtimes-monitoring/index.adoc

```diff
@@ -24,20 +24,24 @@ icon:bullhorn[size=2x]{nbsp} To display custom dashboards, you must set the _app
 
 == Prometheus Configuration
 
-Kiali runtimes monitoring feature works exclusively with Prometheus, so it must be configured correctly to pull your application metrics.
+Kiali custom dashboards work exclusively with Prometheus, so it must be configured correctly to pull your application metrics.
 
 If you are using the default Istio installation, your Prometheus instance should already be configured as shown below and you can skip to the next section.
 
 But if you want to use another Prometheus instance, please refer to link:https://prometheus.io/docs/prometheus/latest/configuration/configuration/#kubernetes_sd_config[Prometheus documentation] to setup `kubernetes_sd_config` for pods. As a reference, link:https://github.com/istio/istio/blob/907aa731c3f76ad21faac98614751e8ab3531893/install/kubernetes/helm/istio/charts/prometheus/templates/configmap.yaml#L229[here is] how it is configured in Istio.
 
-It is important to preserve label mapping, so that Kiali can filter on _app_ and _version_, which is done with this configuration:
+It is important to preserve label mapping, so that Kiali can filter on _app_ and _version_, and to have the _namespace_ label. See the `relabel_configs` that would allow this:
```

Comment:
Just a note that app and version labels on istio metrics are deprecated as they move to canonical_service and canonical_revision.

Reply:
These metrics are not the istio ones, so I don't think it makes any difference here. It's custom metrics and the labels are entirely dependent on prometheus config.

However Istio 1.6 will introduce new built-in possibilities for prometheus custom scraping, I will investigate if it makes sense to change our expected settings, but at the moment I think it's still fine to continue with standard app/version labels.

Reply:
Actually it's not 100% accurate to say it's app or version ; it's mapped after kubernetes labels, and we look for app/version by default but it can be overridden in Kiali config.
I haven't been to that level of details in the doc, but perhaps I should.

Reply:
@jshaughn I have rewritten that part, do you think it's ok?

Reply:
Yes, it's ok.